integration/dubbo-alibaba/src/main/java/io/seata/integration/dubbo/alibaba/AlibabaDubboTransactionPropagationFilter.java

```diff
@@ -43,45 +45,51 @@ public Result invoke(Invoker<?> invoker, Invocation invocation) throws RpcExcept
             return invoker.invoke(invocation);
         }
         String xid = RootContext.getXID();
-        String xidInterceptorType = RootContext.getXIDInterceptorType();
+        String branchType = RootContext.getBranchType();
 
         String rpcXid = getRpcXid();
-        String rpcXidInterceptorType = RpcContext.getContext().getAttachment(RootContext.KEY_XID_INTERCEPTOR_TYPE);
+        String rpcBranchType = RpcContext.getContext().getAttachment(RootContext.KEY_BRANCH_TYPE);
         if (LOGGER.isDebugEnabled()) {
             LOGGER.debug("xid in RootContext[{}] xid in RpcContext[{}]", xid, rpcXid);
         }
         boolean bind = false;
         if (xid != null) {
             RpcContext.getContext().setAttachment(RootContext.KEY_XID, xid);
-            RpcContext.getContext().setAttachment(RootContext.KEY_XID_INTERCEPTOR_TYPE, xidInterceptorType);
+            RpcContext.getContext().setAttachment(RootContext.KEY_BRANCH_TYPE, branchType);
```

Comment:
Is it better to separate and judge?

Reply:
It will not be processed this time, and transaction context object encapsulation will be provided later to handle log output uniformly.